classes/index.js

```diff
@@ -0,0 +1,13 @@
+export * from './Command.js';
+export * from './CommunityPoll.js';
+export * from './LeylineUser.js';
+export * from './Logger.js';
+export * from './collectors/ReactionCollectorBase.js';
+export * from './collectors/GoodActsReactionCollector.js';
+export * from './collectors/KindWordsReactionCollector.js';
+export * from './collectors/ReactionCollector.js';
+export * from './components/ConfirmInteraction.js';
+export * from './components/EmbedBase.js';
+export * from './events/DiscordEvent.js';
+export * from './events/FirebaseEvent.js';
+export * from './services/XPService.js';
```

Comment:
This is a point of interest for me. It might have to be cleaned up in the future, but I liked having all the classes be exported from one directory... makes the import process both easier & cleaner